tests/cloner_test.go

```diff
@@ -94,9 +96,9 @@ func doCloneTest(f *framework.Framework, targetNs *v1.Namespace) {
 		return srcNode == tgtNode
 	}, cloneCompleteTimeout, assertionPollInterval).Should(BeTrue())
 
-	err = f.WaitTimeoutForPodStatus(sourcePod.Name, v1.PodSucceeded, cloneCompleteTimeout)
+	err = f.WaitTimeoutForPodStatus(sourcePod.Name, v1.PodPending, shortTimeout)
```

Comment:
It looks like there are some timing differences between the CI system and my workstation here such that we don't make this check before the delete grace period has completed.  I'll swap this to a wait for delete and we should be in good shape, I'll also include a comment explaining what's going on.